course_info/views.py

```diff
@@ -31,6 +31,8 @@
 ]
 _REFERER_COURSE_ID_RE = re.compile('^.+/courses/(?P<canvas_course_id>\d+)(?:$|.+$)')
 
+_INSTRUCTORS_DISPLAY_FIELD_NAME = "instructors_display"
```

Comment:
Perhaps this should be set above `_FIELD_LABEL_MAP` and used in that dict, so it stays DRY.